pytition/petition/static/petition.css

```diff
@@ -105,7 +127,7 @@ h1 {
     font-size: 1.7em
 }
 
-@media screen and (max-width:768px) {
```

Comment:
Just out of curiosity: why?

Reply:
Because breakpoint is 768px, and small screen width is strictly inferior this breakpoint.